CanvasPlusPlayground/Views/CourseAnnouncementsView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  CourseAnnouncementsView.swift
+//  CanvasPlusPlayground
+//
+//  Created by Ethan Fox on 9/13/24.
+//
+
+import SwiftUI
+
+struct CourseAnnouncementsView: View {
+    let course: Course
+    @State private var announcementManager: CourseAnnouncementManager
+    
+    init(course: Course) {
+        self.course = course
+        self.announcementManager = CourseAnnouncementManager(courseId: course.id)
+    }
+    
+    
+    var body: some View {
+        ZStack {
```

Comment:
nit: could use an `overlay` with a `ContentUnavailableView` instead of `ZStack`, but not imp for this playground app.